stock_move_entries/model/stock_move_entries.py

```diff
@@ -30,11 +30,19 @@
 class AccountMoveLine(osv.Model):
     _inherit = "account.move.line"
 
-    """
-    """
+    def _auto_init(self, cr, context=None):
+        res = super(AccountMoveLine, self)._auto_init(cr, context=context)
+        index = 'account_move_line_sm_id_account_id_index'
+        cr.execute("SELECT indexname FROM pg_indexes WHERE indexname = %s",
+                   (index,))
+        if not cr.fetchone():
+            cr.execute(
+                'CREATE INDEX %s ON account_move_line (account_id, sm_id)' %
```

Comment:
@moylop260  from postgres 9.5 it was added the option IF NOT EXISTS, you can avoid using the first select to verify if this doesn't exist.
```python
cr.execute(
    'CREATE INDEX IF NOT EXISTS %s ON account_move_line (account_id, sm_id)' % index)
```
More about that [here](https://www.postgresql.org/docs/9.5/static/sql-createindex.html)